lib/response.js

```diff
@@ -1130,6 +1133,43 @@ function sendfile(res, file, options, callback) {
   file.pipe(res);
 }
 
+
+function convertBigIntsToNumbers(obj) {
+  // If the type is a BigInt, convert them to number
+  if (typeof obj === 'bigint') {
+    return Number(obj);
+  }
+
+  // If the input is not an object or array, return it as is
+  if (typeof obj !== 'object' || obj === null) {
+    return obj;
+  }
+
+  // If it's an array, loop over its elements and convert them recursively
+  if (Array.isArray(obj)) {
+    for (var index = 0; index < obj.length; index += 1) {
+      obj[index] = convertBigIntsToNumbers(obj[index]);
```

Comment:
Will this mutate the array the user passed into Express, or is this operating on a copy?